<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>KEY:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="inBox">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>31</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="On_insert">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>insert</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="On_remove">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>remove</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="On_search">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_search_min">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>search min key</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="On_print">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>print</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_LoadFactor">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Load Factor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="styleSheet">
           <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    background-color: #FF0000;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_Fill">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Fill the Hash Table</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>501</width>
          <height>481</height>
         </size>
        </property>
        <property name="maxLength">
         <number>300</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
